libs/external-router/routing/src/errors/client-error.ts

```diff
@@ -5,6 +5,8 @@ import {
 
 /**
  * An error thrown when the driver encounters a 4xx error.
+ *
+ * @inheritdoc
```

Comment:
I don't think the errors need to inherit docs, @damienwebdev do you agree?

Reply:
I think I'm just going to remove it, because right now the `@inheritdoc` will only work if the fields in the interface also exist in the class, which isn't the case with all of these errors. They ultimately extend from `Error`